docs/readmes/lte/upgrade_1_6.md

```diff
@@ -0,0 +1,37 @@
+---
+id: upgrade_1_6
+title: Upgrade to v1.6
+hide_title: true
+---
+
+# Upgrade to v1.6
+
+### Fresh Install Notes
+
+For fresh gateway installs, please use the following install scripts.
+
+```bash
+wget https://raw.githubusercontent.com/magma/magma/master/lte/gateway/deploy/agw_install_ubuntu.sh
+chmod +x agw_install_ubuntu.sh
+./agw_install_ubuntu.sh
+```
+
+### Upgrading from previous releases
+
+Beginning with Magma Fuji (v1.5), Magma artifacts are now hosted on the new
+Magmacore repositories. Gateways migrating from older Magma releases should
+run the migration script
+(https://raw.githubusercontent.com/magma/magma/master/lte/gateway/release/upgrade_magma.sh)
+to update the sources accordingly.
+
+To upgrade an existing AGW, please run the following upgrade script.
+
+```bash
+wget https://raw.githubusercontent.com/magma/magma/master/lte/gateway/release/upgrade_magma.sh
+chmod +x upgrade_magma.sh
+./upgrade_magma_sh
+```
+
+### Image Version
+
+Ubuntu > ``
```

Comment:
?

Reply:
I'm waiting for the magma pkg version which would be promoted and will update this